contracts/guard/src/GuardV0.sol

```diff
@@ -58,6 +74,9 @@ contract GuardV0 is IGuard, Ownable {
     // Allowed routers
     mapping(address destination => bool allowed) public allowedApprovalDestinations;
 
+    // Allowed routers
```

Comment:
Maybe needs a bit more context what is delegation approval - link to some info about it